### PR TITLE
Updated all large area tools to use agent properties by default while…

### PR DESCRIPTION
**Pull request by KieranRatcliffeInvertedAI**
… maintaining backwards compatibility. 

This will be an official fix for the large simulation stack that was temporarily fixed before in an unmerged branch. This has been tested by passing both `AgentProperties` and `AgentAttributes` into the `large_map_example.py` script into both the `Regions` and the `large_drive` function as the two main access points where it could be either data type. 

If you find or think of anywhere else this change should be made (except other examples since that will be another branch), please let me know.

### CODE REVIEW

**Review by adscib — COMMENTED** [review]
I don't see anything suspicious here, but I'll let the others review it.

**Comment by KieranRatcliffeInvertedAI** [comment]
@rf-ivtdai @Ruishenl pinging you two for feedback on this PR.

File: invertedai/utils.py
Comment (rf-ivtdai) [inline invertedai/utils.py:448]: If this is meant to be a flag to by default use or not use `agent_properties`, change it to boolean `use_agent_properties` with a default value. Also update the doc string below accordingly.

File: invertedai/large/common.py
Comment (rf-ivtdai) [inline invertedai/large/common.py:100]: It seems this method could be called in a few other places as well. Consider moving it to utils and call it everywhere you need to convert attributes into properties.

File: invertedai/large/drive.py
Comment (rf-ivtdai) [inline invertedai/large/drive.py:97]: call `convert_attributes_to_properties`

File: invertedai/utils.py
Comment (KieranRatcliffeInvertedAI) [reply invertedai/utils.py]: Done

File: invertedai/large/common.py
Comment (KieranRatcliffeInvertedAI) [reply invertedai/large/common.py]: Done.

**Comment by KieranRatcliffeInvertedAI** [comment]
@rf-ivtdai if the changes I've made from your suggestions look good, is there anything else to change before merging this branch?

**Review by Ruishenl — APPROVED** [review]

**Comment by rf-ivtdai** [comment]
> @rf-ivtdai if the changes I've made from your suggestions look good, is there anything else to change before merging this branch?

Looks good :)

**Review by rf-ivtdai — APPROVED** [review]

**Pull request merged**
